chitter.rb

```diff
@@ -0,0 +1,57 @@
+require 'sinatra/base'
+require './lib/message.rb'
+require './lib/user.rb'
+require 'pg'
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+    
+  get '/' do
+    redirect('/messages')
+  end
+
+  get '/messages' do
+    @list = Message.all
+    @user = session[:user]
+    #@authors = @messages.each { |message| message.author }
+    erb :message_board
+  end
+
+  post '/messages/new' do
```

Comment:
As pwer rubric - may be helpful to split controller into seperate files: https://github.com/squareben1/chitter-challenge/blob/master/docs/review.md#split-routes-into-separate-controller-files